walk-in-interview/src/main/java/com/google/job/data/JobStatus.java

```diff
@@ -0,0 +1,34 @@
+package com.google.job.data;
+
+/** Enumeration that represents the status of a job post. */
+public enum JobStatus {
+    ACTIVE("ACTIVE"),
+    DELETED("DELETED"),
+    EXPIRED("EXPIRED");
+
+    private final String statusId;
+
+    JobStatus(String statusId) {
+        this.statusId = statusId;
+    }
+
+    /**
+     * Gets the id string.
```

Comment:
nit, `@return` is superfluous, perfer this:
```
/** Returns the stable id representing the job status. Can be stored in database. */
```

Reply:
Fixed